nistats/tests/test_first_level_model.py

```diff
@@ -382,3 +382,28 @@ def test_first_level_models_from_bids():
         # can arise when variant or space is present and not specified
         assert_raises(ValueError, first_level_models_from_bids,
                       bids_path, 'main', 'T1w')  # variant not specified
+
+def test_first_level_models_with_no_signal_scaling():
+    """
+    test to ensure that the FirstLevelModel works correctly with a
+    signal_scaling==False. In particular, that derived theta are correct for a
+    constant design matrix with a single valued fmri image
+    """
+    shapes, rk = [(3, 1, 1, 2)], 1
+    fmri_data = []
+    design_matrices = []
+    design_matrices.append( pd.DataFrame(np.ones((shapes[0][-1], rk) ),
+                                columns=list('abcdefghijklmnopqrstuvwxyz')[:rk]))
+    first_level_model = FirstLevelModel(mask=False, noise_model='ols', signal_scaling=False)
+    fmri_data.append(Nifti1Image( np.zeros((1,1,1,2)) + 6, np.eye(4)))
+
+    first_level_model.fit(fmri_data, design_matrices=design_matrices)
+    # trivial test of signal_scaling value
+    assert_true(first_level_model.signal_scaling==False)
+    # assert that our design matrix has one constant
+    assert_true(first_level_model.design_matrices_[0].equals(
+                    pd.DataFrame([1.0, 1.0], columns=['a'])))
+    # assert that we only have one theta as there is only on voxel in our image
+    assert_true(first_level_model.results_[0][0].theta.shape == (1,1))
+    # assert that the theta is equal to the one voxel value
+    assert_almost_equal(first_level_model.results_[0][0].theta[0,0],6.0,2)
```

Comment:
The test is great. I believe that you could have made it more elementary, i.e. testing only the function you've changed, but that's fine.
There are some PEP8 issues (space around operators). Can you fix them ?
Thx again !